shared/scripts/test-ganache.sh

```diff
@@ -19,7 +19,7 @@ cleanup() {
 setup_testing_variables() {
   PORT=${PORT-8545}
   BALANCE=${BALANCE-100000}
-  GAS_LIMIT=${GAS_LIMIT-8000000}
+  GAS_LIMIT=${GAS_LIMIT-10000000}
```

Comment:
We shouldn't be working with non-real gas values. If we are hitting a gas limit, we should split the transactions